solana-programs/start.sh

```diff
@@ -142,33 +142,33 @@
     yarn run ts-node cli/main.ts --function initAdmin --owner-keypair $owner_wallet_path --network $SOLANA_HOST
 
     # Propagate local variables
-    admin_keypair_path="$PWD/adminKeypair.json"
-    admin_storage_keypair_path="$PWD/adminStorageKeypair.json"
-    admin_keypair_publickey=$(solana-keygen pubkey adminKeypair.json)
-    admin_keypair_privatekey=$(cat adminKeypair.json)
-    admin_storage_keypair_publickey=$(solana-keygen pubkey adminStorageKeypair.json)
-    admin_storage_keypair_privatekey=$(cat adminStorageKeypair.json)
+    admin_authority_keypair_path="$PWD/adminAuthorityKeypair.json"
+    admin_storage_keypair_path="$PWD/adminAccountKeypair.json"
+    admin_authority_keypair_publickey=$(solana-keygen pubkey adminAuthorityKeypair.json)
+    admin_authority_keypair_privatekey=$(cat adminAuthorityKeypair.json)
+    admin_account_keypair_publickey=$(solana-keygen pubkey adminAccountKeypair.json)
+    admin_storage_keypair_privatekey=$(cat adminAccountKeypair.json)
 
     # initialize Content/URSM nodes - initContentNode uses deterministic 
     # addresses and pkeys from eth-contracts ganache chain.
     yarn run ts-node cli/main.ts --function initContentNode \
         --owner-keypair "$owner_wallet_path" \
-        --admin-keypair "$admin_keypair_path"\
-        --admin-storage-keypair "$admin_storage_keypair_path" \
+        --admin-authority-keypair "$admin_authority_keypair_path"\
```

Comment:
this is the main fix. using the right path because adminKeypair.json shouldn't exist anymore.